apps/docs/docs/modules/data_loaders/llama_parse/index.mdx

```diff
@@ -41,7 +41,9 @@ They can be divided into two groups.
 - `doNotCache?` Optional. Set to true to not cache the document.
 - `fastMode?` Optional. Set to true to use the fast mode. This mode will skip OCR of images, and table/heading reconstruction. Note: Non-compatible with `gpt4oMode`.
 - `doNotUnrollColumns?` Optional. Set to true to keep the text according to document layout. Reduce reconstruction accuracy, and LLMs/embeddings performances in most cases.
-- `pageSeparator?` Optional. The page separator to use. Defaults is `\\n---\\n`.
+- `pageSeparator?` Optional. A templated page separator to use to split the text. If it contain `{page_number}`, it will be replaced by the next page number. If not set the default separator `\\n---\\n` will be used.
```

Comment:
Why does the `pageSeparator` contain the `{page_number}`?

Reply:
Sorry, I just copied the python description. 
I assume it's currently only relevant for JSON mode where a page_number is returned.
Improved the descriptions.